src/products/components/ProductUpdatePage/ProductUpdatePage.tsx

```diff
@@ -169,6 +188,24 @@ export const ProductUpdatePage: React.FC<ProductUpdatePageProps> = ({
       value: taxType.taxCode
     })) || [];
 
+  const openAssignReferencesAttributeDialog = !!assignReferencesAttributeId;
```

Comment:
I'm not convinced that it's a good name... how about `canOpenAssignReferencesAttributeDialog`?

Reply:
fixed